Platform/CommonBoardPkg/CfgData/Template_BootOption.dsc

```diff
@@ -6,17 +6,22 @@
 ##
 
   #
-  # ========================= OS_TMPL ==========================
+  # ========================= BOOT_OPTION_TMPL ==========================
   #
   # NOTE: MUST match OS_BOOT_OPTION structure!!!
-  # !BSF DEFT:{OS_TMPL:START}
+  # !BSF DEFT:{BOOT_OPTION_TMPL:START}
     # !BSF PAGES:{OS_$(1):OS:"Boot Option $(1)"}
     # !BSF PAGE:{OS_$(1)}
 
+    # !BSF SUBT:{CFGHDR_TMPL:BOOT_OPTION_CFG_DATA_$(1):0:0}
+    # !HDR EMBED:{BOOT_OPTION_CFG_DATA_$(1):TAG_05$(1):START}
+
     # !BSF NAME:{Image Type}
     # !BSF TYPE:{Combo}
-    # !BSF OPTION:{0:Default, 1:Android, 2:ClearLinux, 3:Acrn, 4:Fastboot}
-    # !BSF HELP:{Specify boot image type}
+    # !BSF OPTION:{0:Default, 1:Android, 2:ClearLinux, 3:Acrn, 4:Fastboot, 0xFE:Addendum, 0xFF:Not used}
+    # !BSF HELP:{Specify boot image type.}
```

Comment:
wondering what user should set if they want to create a boot option for Android with A/B support. It is Android image, and also need addendum to add boot option information for misc partition.

Reply:
It should be marked as Android image, and then in the Flags, it should mark A/B support.  
An addendum entry will be appended to provide misc partition info.

Reply:
Understood.  If one boot option (e.g. boot option 1) has addendum, will the addendum been shown as a separate boot option (e.g. boot option 2) or as an addendum together with its primary boot option (e.g. boot option 1) in ConfigEditor?

Reply:
In ConfigEditor, it will be showed as a boot option, but with Image Type set to "Addendum"

Reply:
I remember previously it will shows "boot option 1", "boot option 2" for every boot option entry. 
If we still have these information there, it would be a little confusing since "boot option 2" is really boot option 2, basically it is part of boot option 1.
Not sure if we could show one full boot option in same page as before.

Reply:
It won't be able to show everything in a single page like before.  Here the entry might not equal to boot option. since multiple entries can be combined.   We can hide some items to let user know it is not a normal option.

Reply:
Yes, better hide the things not necessary for addendum.
and still prefer not show boot option 2 for the addendum for option 1.  Need update "OS:"Boot Option $(1)""

Reply:
Added code to hide some items for addendum entry.
But regarding to not show boot option 2, I don't believe  it is feasible. 
Even if you do so,  if somebody config the image type back to normal type, it will become a normal option again and should not hide.  Let us keep it that way.   And considering people will use the normal boot option entry in most of the time, I believe it is OK.

Reply:
Understand the title is static, not support dynamic change now. so let's use it now.